### ISSUE TITLE
Can't upgrade django-concurrency to 1.0

### ISSUE THREAD

**Issue by oppianmatt** [saxix/django-concurrency#56]
```
Collecting django-concurrency==1.0 (from -r requirements.txt (line 41))
  Using cached django-concurrency-1.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-build-pMvY07/django-concurrency/setup.py", line 83, in <module>
        name=app.NAME,
    AttributeError: 'module' object has no attribute 'NAME'
    
    ----------------------------------------

Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-pMvY07/django-concurrency
```

Trying to upgrade django-concurrency but it's failing. Only django concurrency has this issue, all the other packages are fine.

**Comment by oppianmatt** [comment]
Investigated the issue, setup.py is making the assumption that `concurrency/__init__.py` has a `NAME` property. In version 0.8.1-final-0 it does not (the version that happens to be on the server). This changelist:

https://github.com/saxix/django-concurrency/commit/423a0336f6e6ca894a11790a3504af28de2cec23

Have edited to title to reflect the issue.

**Comment by saxix** [comment]
just released  [1.0.1](https://pypi.python.org/pypi?:action=display&name=django-concurrency&version=1.0.1).
